cmd/fossa/init.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+
+	logging "github.com/op/go-logging"
+	"github.com/urfave/cli"
+
+	"github.com/fossas/fossa-cli/builders"
+	config "github.com/fossas/fossa-cli/config"
+)
+
+var initLogger = logging.MustGetLogger("init")
+
+func initCmd(c *cli.Context) {
+	conf, err := config.New(c)
+	if err != nil {
+		initLogger.Fatalf("Could not load configuration: %s", err.Error())
+	}
+
+	if err := doInit(&conf, c.Bool("force"), c.Bool("include-all")); err != nil {
+		initLogger.Fatalf("Error initializing: %s", err.Error())
+	}
+
+	if !c.Bool("skip-write") {
+		if err := config.WriteConfigFile(&conf); err != nil {
+			initLogger.Fatalf("Error writing config: %s", err.Error())
+		}
+
+		initLogger.Warningf("Config for %d modules written to `%s`.", len(conf.Modules), conf.ConfigFilePath)
+	}
+
+	fmt.Println("`fossa` is initialized")
+}
+
+func doInit(conf *config.CliConfig, force bool, includeAll bool) error {
+	var err error
+	if len(conf.Modules) == 0 || force {
+		conf.Modules, err = findModules(".")
```

Comment:
You should pass an absolute path instead, for robustness. (Personally, I'd use a type alias to ensure you use an absolute path.)